### DISCUSSION TITLE
Option "Warnings" -> "Saving projects" (and "Saving empty project") is not working

### DISCUSSION THREAD

**Discussion by galdralag** [audacity/audacity#4315] [Q&A]
### Bug description

Audacity always asks for saving project.
Option "Warnings" -> "Saving projects" (and "Saving empty project") has no effect.

### Steps to reproduce

- in preferences dialog enable options "Warnings" -> "Saving projects" and "Saving empty project"
- open mp3 file
- try close audacity
- you will be prompted for saving project

### Expected behavior

When "Saving projects" and "Saving empty project" options enabled audacity should never ask for saving projects

### Audacity Version

stable version (from a Linux repository)

### Operating system

Linux

**Comment by LWinterberg** [comment]
This is not what these settings control, see https://manual.audacityteam.org/man/warnings_preferences.html
